templates/dashboard/program-edit.html

```diff
@@ -207,10 +209,11 @@ <h5 class="modal-title" id="{{ description_name|slugify }}-label">{{ description
     });
   </script>
   <script>
-    var JOBS_TYPEAHEAD_URL = "{% url 'api.jobs.list' %}";
+    var JOBS_TAGIFY_URL = "{% url 'api.jobs.list' %}";
   </script>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-tokenfield/0.12.0/bootstrap-tokenfield.min.js" integrity="sha256-jdwX0QzXB7z7Xc7Vz0ovtIHWO5qIZWg0aLcGv44JDgE=" crossorigin="anonymous"></script>
```

Comment:
Can we remove this if it is no longer necessary?

Reply:
Yes We could remove it.